src/test/java/seedu/address/logic/comparator/ApplicationDateThenInterviewDateComparatorTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.comparator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.internship.Address;
+import seedu.address.model.internship.ApplicationDate;
+import seedu.address.model.internship.InternshipApplication;
+import seedu.address.model.internship.interview.Interview;
+import seedu.address.testutil.InternshipApplicationBuilder;
+
+class ApplicationDateThenInterviewDateComparatorTest {
+
+    @Test
+    public void compare() {
+        InternshipApplicationBuilder internship1 = new InternshipApplicationBuilder();
+        internship1.withApplicationDate(new ApplicationDate(LocalDate.now()));
```

Comment:
Maybe a good idea to declare todaydate = localdate.now() at the start? Then tests wont fail at midnight lolol.

Reply:
I personally dont like the idea of using LocalDate.now() everywhere cause its like a static method.

Reply:
True! Thanks!